test/debug/bp_test.rb

```diff
@@ -50,5 +50,13 @@ def test_breakpoint_execute_command_argument_correctly
         type 'q!'
       end
     end
+
+    def test_debugger_doesnt_complain_about_duplicated_breakpoint
+      debug_code(program) do
+        type 'continue'
+        assert_no_line_text(/duplicated breakpoint:/)
```

Comment:
currently, writing multiple `binding.bp` triggers this warning. but the PR also fixes this.